docs/index.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>DAPI-Client - Client library used to access Dash DAPI endpoints</title>
+    <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1" />
+    <meta name="keywords" content="dapi-client, dapi, dash, javascript">
+    <meta name="description" content="Client library used to access Dash DAPI endpoints">
+    <meta name="viewport" content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <link rel="stylesheet" href="//unpkg.com/docsify/lib/themes/vue.css">
+</head>
+<body>
+<div id="app"></div>
+<script>
+  window.$docsify = {
+    name: 'DAPI-Client',
+    repo: 'https://github.com/dashevo/dapi-client',
+    loadSidebar: "_sidebar.md",
+    maxLevel: 4,
+    subMaxLevel: 2,
+    search: {
+      noData: {
+        '/': 'No results!'
+      },
+      paths: 'auto',
+      placeholder: {
+        '/': 'Search'
+      }
+    },
+    // search: 'auto',
+    formatUpdated: '{MM}/{DD} {HH}:{mm}',
```

Comment:
Add Dash blue theme color
```suggestion
    formatUpdated: '{MM}/{DD} {HH}:{mm}',
    themeColor: '#008de4',
```